packages/tooling/fast-tooling/src/message-system/message-system.utilities.props.ts

```diff
@@ -46,6 +46,10 @@ export enum MessageSystemHistoryTypeAction {
     get = "get",
 }
 
+export enum MessageSystemSchemaDictionaryTypeAction {
```

Comment:
Favor types over enums for perf.

Reply:
This is already the standard format for describing messages for the message system.